Jenkinsfile

```diff
@@ -17,17 +20,27 @@ pipeline {
                    sh 'docker login -u yahya4246 -p ${dockerhubpwd}'
 
 }
-                   sh 'docker push yahya4246/jenkins-image'
+                   sh 'docker push yahya4246/jenkins-image:${IMAGE_TAG}'
```

Comment:
Add Docker image as tag as well in enviroment 
DOCKER_REPO = "yahya4246/jenkins-image"

Reply:
done